### PR TITLE
YARN-11362: Fix several typos in YARN codebase of misspelled resource

### PR DESCRIPTION
**Pull request by EremenkoValentin**
### Description of PR

JIRA: YARN-11362: Fix several typos in YARN codebase of misspelled resource.

### For code changes:

- [x] Does the title or this PR starts with the corresponding JIRA issue id (e.g. 'HADOOP-17799. Your PR title ...')?
- [ ] Object storage: have the integration tests been executed and the endpoint declared according to the connector-specific documentation?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the `LICENSE`, `LICENSE-binary`, `NOTICE-binary` files?

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 21s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  |  The patch appears to include 5 new or modified test files.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  13m 58s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  22m  1s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   3m 38s |  |  trunk passed with JDK Ubuntu-11.0.21+9-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  compile  |   3m 45s |  |  trunk passed with JDK Private Build-1.8.0_392-8u392-ga-1~20.04-b08  |
| +1 :green_heart: |  checkstyle  |   1m  5s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   2m 41s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   2m 34s |  |  trunk passed with JDK Ubuntu-11.0.21+9-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   2m 37s |  |  trunk passed with JDK Private Build-1.8.0_392-8u392-ga-1~20.04-b08  |
| +1 :green_heart: |  spotbugs  |   4m 57s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  21m 29s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 20s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   1m 37s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 18s |  |  the patch passed with JDK Ubuntu-11.0.21+9-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javac  |   3m 18s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 16s |  |  the patch passed with JDK Private Build-1.8.0_392-8u392-ga-1~20.04-b08  |
| +1 :green_heart: |  javac  |   3m 16s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  checkstyle  |   0m 54s |  |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   2m 17s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   2m 14s |  |  the patch passed with JDK Ubuntu-11.0.21+9-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   2m 24s |  |  the patch passed with JDK Private Build-1.8.0_392-8u392-ga-1~20.04-b08  |
| +1 :green_heart: |  spotbugs  |   5m 10s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  19m 26s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 44s |  |  hadoop-yarn-api in the patch passed.  |
| +1 :green_heart: |  unit  |   4m 35s |  |  hadoop-yarn-common in the patch passed.  |
| +1 :green_heart: |  unit  |   3m 23s |  |  hadoop-yarn-server-applicationhistoryservice in the patch passed.  |
| +1 :green_heart: |  unit  |  86m 23s |  |  hadoop-yarn-server-resourcemanager in the patch passed.  |
| +1 :green_heart: |  unit  |  25m 27s |  |  hadoop-yarn-client in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 37s |  |  The patch does not generate ASF License warnings.  |
|  |   | 245m 41s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.44 ServerAPI=1.44 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-6474/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/6474 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient spotbugs checkstyle codespell detsecrets |
| uname | Linux f54497408569 5.15.0-88-generic #98-Ubuntu SMP Mon Oct 2 15:18:56 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 629f18be1f01ef5ba6b355b6c21fd93d9e035b9d |
| Default Java | Private Build-1.8.0_392-8u392-ga-1~20.04-b08 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.21+9-post-Ubuntu-0ubuntu120.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_392-8u392-ga-1~20.04-b08 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-6474/1/testReport/ |
| Max. process+thread count | 2553 (vs. ulimit of 5500) |
| modules | C: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client U: hadoop-yarn-project/hadoop-yarn |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-6474/1/console |
| versions | git=2.25.1 maven=3.6.3 spotbugs=4.2.2 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by EremenkoValentin** [comment]
Hello everyone. Despite the fact that Jira was created a long time ago and an assignee was appointed, the issue still hasn't been fixed. Could someone please review my PR?

**Comment by slfan1989** [comment]
> Hello everyone. Despite the fact that Jira was created a long time ago and an assignee was appointed, the issue still hasn't been fixed. Could someone please review my PR?

LGTM.

**Review by slfan1989 — APPROVED** [review]

**Comment by slfan1989** [comment]
@ayushtkn Can you help review this PR? Thank you very much!

**Comment by EremenkoValentin** [comment]
> @ayushtkn Can you help review this PR? Thank you very much!

Up

**Comment by EremenkoValentin** [comment]
Hello, @ayushtkn. Could you, please, help review this PR? Thank you very much!

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/exceptions/SchedulerInvalidResourceRequestException.java [hunk 1]
```diff
@@ -29,19 +29,19 @@
...
 
-  public SchedulerInvalidResoureRequestException(Throwable cause) {
+  public SchedulerInvalidResourceRequestException(Throwable cause) {
     super(cause);
   }
 
-  public SchedulerInvalidResoureRequestException(String message,
-      Throwable cause) {
+  public SchedulerInvalidResourceRequestException(String message,
+                                                  Throwable cause) {
```
Comment (slfan1989) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/exceptions/SchedulerInvalidResourceRequestException.java]: We need to pay attention to the indentation, usually 5 characters.

**Review by slfan1989 — APPROVED** [review]

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/exceptions/SchedulerInvalidResourceRequestException.java (hunk 1 above)
Comment (EremenkoValentin) [reply hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/exceptions/SchedulerInvalidResourceRequestException.java]: Thanks for your review. Corrected

**Comment by hadoop-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 22s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  |  The patch appears to include 5 new or modified test files.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  14m 19s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  19m 48s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   3m 30s |  |  trunk passed with JDK Ubuntu-11.0.21+9-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  compile  |   3m 17s |  |  trunk passed with JDK Private Build-1.8.0_392-8u392-ga-1~20.04-b08  |
| +1 :green_heart: |  checkstyle  |   0m 57s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   2m 33s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   2m 27s |  |  trunk passed with JDK Ubuntu-11.0.21+9-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   2m 26s |  |  trunk passed with JDK Private Build-1.8.0_392-8u392-ga-1~20.04-b08  |
| +1 :green_heart: |  spotbugs  |   4m 46s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  19m 48s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 19s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   1m 37s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 12s |  |  the patch passed with JDK Ubuntu-11.0.21+9-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javac  |   3m 12s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 10s |  |  the patch passed with JDK Private Build-1.8.0_392-8u392-ga-1~20.04-b08  |
| +1 :green_heart: |  javac  |   3m 10s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  checkstyle  |   0m 52s |  |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   2m 26s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   2m  8s |  |  the patch passed with JDK Ubuntu-11.0.21+9-post-Ubuntu-0ubuntu120.04  |
| +1 :green_heart: |  javadoc  |   2m 26s |  |  the patch passed with JDK Private Build-1.8.0_392-8u392-ga-1~20.04-b08  |
| +1 :green_heart: |  spotbugs  |   5m 19s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  19m 41s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 42s |  |  hadoop-yarn-api in the patch passed.  |
| +1 :green_heart: |  unit  |   4m 35s |  |  hadoop-yarn-common in the patch passed.  |
| +1 :green_heart: |  unit  |   3m 18s |  |  hadoop-yarn-server-applicationhistoryservice in the patch passed.  |
| +1 :green_heart: |  unit  |  86m 11s |  |  hadoop-yarn-server-resourcemanager in the patch passed.  |
| +1 :green_heart: |  unit  |  25m 24s |  |  hadoop-yarn-client in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 34s |  |  The patch does not generate ASF License warnings.  |
|  |   | 240m  9s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.44 ServerAPI=1.44 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-6474/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/6474 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient spotbugs checkstyle codespell detsecrets |
| uname | Linux 607eac38a085 5.15.0-88-generic #98-Ubuntu SMP Mon Oct 2 15:18:56 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 220fcccc46405413647d4acdf7d9b9114971fb53 |
| Default Java | Private Build-1.8.0_392-8u392-ga-1~20.04-b08 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.21+9-post-Ubuntu-0ubuntu120.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_392-8u392-ga-1~20.04-b08 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-6474/2/testReport/ |
| Max. process+thread count | 2775 (vs. ulimit of 5500) |
| modules | C: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client U: hadoop-yarn-project/hadoop-yarn |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-6474/2/console |
| versions | git=2.25.1 maven=3.6.3 spotbugs=4.2.2 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by slfan1989** [comment]
@EremenkoValentin  Thanks for the contribution! I will merge this pr to the trunk branch in 1-2 days.

**Comment by slfan1989** [comment]
@EremenkoValentin Thanks for the contribution, merged to trunk!  Do you have a JIRA account?

**Comment by EremenkoValentin** [comment]
> @EremenkoValentin Thanks for the contribution, merged to trunk! Do you have a JIRA account?

Thanks @slfan1989 for your review and merge. I submitted a request to create an account but still have not received a response.

**Comment by slfan1989** [comment]
> Thanks @slfan1989 for your review and merge. I submitted a request to create an account but still have not received a response.

@EremenkoValentin your JIRA account has been created, Welcome to hadoop!

**Pull request merged**